testing/integration/upgrade_standalone_same_commit_test.go

```diff
@@ -74,7 +74,7 @@ func TestStandaloneUpgradeSameCommit(t *testing.T) {
 			upgradetest.WithUnprivileged(unprivilegedAvailable),
 			upgradetest.WithDisableHashCheck(true),
 		)
-		assert.ErrorContainsf(t, err, fmt.Sprintf("agent version is already %s", currentVersion), "upgrade should fail indicating we are already at the same version")
+		assert.NoError(t, err)
```

Comment:
Please add a message to explain why we expect no error in this case (otherwise this test is functionally equal to the next one in line 80)